getting-started/openshift/windows/installation.md

```diff
@@ -210,15 +210,21 @@ $ ./wni aws create \
 
 #### Configure kubelet
 
-Copy the previously downloaded file `wmcb.exe` and your worker's ignition file (worker.ign) to the Windows node.
-To download the worker.ign:
+Copy the previously downloaded file `wmcb.exe` to the Windows node.
+From the Windows node, download the worker.ign from an API server pod:
 
-```shell
-apiserver=$(oc get po -n  openshift-kube-apiserver -l apiserver=true --no-headers -o custom-columns=":metadata.name" | head -n 1)
-oc -n openshift-kube-apiserver exec ${apiserver} -- curl -ks https://localhost:22623/config/worker > worker.ign
+```powershell
+$apiServer = c:\k\kubectl --kubeconfig c:\k\config get po -n  openshift-kube-apiserver -l apiserver=true --no-headers -o custom-columns=":metadata.name" | select -first 1
+c:\k\kubectl --kubeconfig c:\k\config -n openshift-kube-apiserver exec $apiserver -- curl -ks https://localhost:22623/config/worker > worker.ign
+```
```

Comment:
I suggest we use absolute path  (c:\worker.ign) ,  so if user ever moved to another directory, things still working.